### ISSUE TITLE
pelican-import claims SITEURL and TIMEZONE are not set, when they actually are.

### ISSUE THREAD

**Issue by stolk** [getpelican/pelican#2953]
- [x] I have read the [Filing Issues](https://docs.getpelican.com/en/latest/contribute.html#filing-issues) and subsequent “How to Get Help” sections of the documentation.
- [x] I have searched the [issues](https://github.com/getpelican/pelican/issues?q=is%3Aissue) (including closed ones) and believe that this is not a duplicate.

- **OS version and name**: Ubuntu 20.04.3
- **Python version**:  3.8.10
- **Pelican version**: 4.7.1

## Issue

When I import a blogger site using pelican-import, I get a warning on TIMEZONE and SITEURL not being set.

But this is incorrect, they **have been set** as can be seen here:

```
(pelican) root@localhost:~/website# grep SITEURL pelicanconf.py 
SITEURL = 'https://stolk.org/blog'
(pelican) root@localhost:~/website# grep TIMEZONE pelicanconf.py 
TIMEZONE = 'America/Vancouver'
(pelican) root@localhost:~/website# pelican-import --blogger ~/blog-11-24-2021.xml 
[00:09:49] WARNING  Feeds generated without SITEURL set properly may not be valid                                                                    log.py:91
           WARNING  No timezone information specified in the settings. Assuming your timezone is UTC for feed generation. Check                      log.py:91
                    https://docs.getpelican.com/en/latest/settings.html#TIMEZONE for more information                                                         
content/bugs-in-complex-software.rst
content/switching.rst
content/bi-colour-led-bars.rst
...
```

My config:
```
AUTHOR = 'Bram Stolk'
SITENAME = 'stolk.org'
SITEURL = 'https://stolk.org/blog'

PATH = 'content'

#THEME = 'pelican-striped-html5up'
THEME = 'clean-blog'

TIMEZONE = 'America/Vancouver'

DEFAULT_LANG = 'en'

# Feed generation is usually not desired when developing
FEED_ALL_ATOM = None
CATEGORY_FEED_ATOM = None
TRANSLATION_FEED_ATOM = None
AUTHOR_FEED_ATOM = None
AUTHOR_FEED_RSS = None

# Blogroll
LINKS = (('Pelican', 'https://getpelican.com/'),
         ('Python.org', 'https://www.python.org/'),
         ('Jinja2', 'https://palletsprojects.com/p/jinja/'),
         ('You can modify those links in your config file', '#'),)

# Social widget
SOCIAL = (('You can add links in your config file', '#'),
          ('Another social link', '#'),)

DEFAULT_PAGINATION = False

# Uncomment following line if you want document-relative URLs when developing
#RELATIVE_URLS = True

```

**Comment by avaris** [comment]
For the record, `pelican-import` doesn't use (or need) the settings file. Warnings seem to be a side effect of not using the proper method to access some defaults. They should be harmless and shouldn't affect any functionality. They shouldn't be outputted, true, but they are nothing more than unnecessary clutter.. just to make it clear.

**Comment by avaris** [comment]
Fixed in #3115
